source/community/release_notes/palm.rst

```diff
@@ -15,14 +15,109 @@ These are the release notes for the Palm release, the 16th community release of
+* Clicking the “Refine” button opens options to filter by course status or sort either by most recent enrollment (default) or title.
+
+.. image:: /_images/community/release_notes/palm/new_learner_home_filtering.png
+    :alt: The Refine pop-up with Course State and Sort options
+
+* All of a user’s courses are fetched on page load. To make the page manageable, we paginate that list, showing 25 courses at a time. To view other courses, a user should page through their list of courses using the pagination controls at the bottom of the course list.
+
+.. image:: /_images/community/release_notes/palm/new_learner_home_pagination.png
+    :alt: Next, previous and page number buttons appear below the list of courses
+
+* Site staff can now masquerade as users on the platform by typing a username or email in the “View as” box and hitting submit. This is designed to be “view only” so change actions (e.g. enroll, unenroll, selecting a session) are blocked in this view.
+* There is a complete list of changes in the `open edX wiki <https://openedx.atlassian.net/wiki/spaces/OEPM/blog/2022/11/21/3584589831/2U+New+Learner+Home+Page#Comparison-with-old-experience>`_. 
+
+To turn on the New Learner home page, add and enable the :code:`learner_home_mfe.enabled` Waffle Flag in the django 
+admin. To support a phased role out, there is also a setting :code:`LEARNER_HOME_MFE_REDIRECT_PERCENTAGE`. Set it to an 
+integer value (0 - 100) representing what percentage of users should be redirected to Learner Home. 0 represents no 
+rollout, 100 represents rollout to all users.
+
 
 Instructor Experiences
 **********************
 
+New Visual Problem Editor
+=========================
+
+The release includes an improved problem-authoring experience with an interactive WYSIWYG editor. Writing markdown 
+code is no longer necessary. But, the advanced mode is still available, maintaining the ability to write and edit 
+OLX XML.
+
+The open edX wiki page `[2U] New Visual Problem Editor <https://openedx.atlassian.net/wiki/spaces/OEPM/blog/2023/04/07/3724312593/2U+New+Visual+Problem+Editor>`_ 
+provides a brief explanation of what has changed in the problem editor. Updated detailed instructions on writing 
+problems can be found in section `8.4. Working with Problem Components <https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/course_components/create_problem.html#working-with-problem-components>`_ of the Building and Running an edX Course documentation.
+
+The visual problem editor is an update to the Course Authoring Micro-frontend first released in Olive. Assuming that 
+you are already using the Course Authoring MFE, all you need to do to enable the Visual Problem Editor is add the 
```

Comment:
How would I know if I was using the Course Authoring MFE? Was that an optional add in Olive? Is it not default/required in palm?

Reply:
Looking back at the Olive release notes, I see that the Course Authoring MFE was enabled by default, so I clarified this.